bucket/vapoursynth.json

```
@@ -0,0 +1,48 @@
+{
+    "version": "R57",
+    "description": "A video processing framework with simplicity in mind ",
+    "homepage": "http://www.vapoursynth.com/",
+    "depends": [
```

```suggestion
    "depends": "extras/vcredist2022"
```
**VCRedist2022** now covers all the functions from VC2015-2022.
https://docs.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170